Portfolio2.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <title>Elmi Farah's Portfolio'</title>
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <link rel="stylesheet"type="text/css"href="portfolio2.css">
+   </head>
+   <body>
+      <header>
+               <ul>
+                  <li class="logo"> <img src="maydenacademylogo.png"/> </li>
+                  <a href="#aboutme"><li class="active"> ABOUT ME </li> </a>
```

Comment:
Just switch the order on these, so that the a tag is inside of the <li> </li>

The only element allowed directly inside of a ul is an li.